### DISCUSSION TITLE
cond for TASK

### DISCUSSION THREAD

**Discussion by petr-klinger** [cybertec-postgresql/pg_timetable#372] [Ideas]
I suggest adding a text variable (task_cond ???) to the TASK table. You can enter an SQL statement of type boolean, which would dynamically determine the task as live (similar to chain.live). Thus, IF-THEN could be implemented between individual tasks of the same chain_id.

**Comment by pashagolub** [comment]
Hello.

Nice feature request. Right now, one can probably add this logic into the task itself.
